client/source/utilities/axiosRequests.js

```diff
@@ -1,7 +1,6 @@
 import axios from 'axios';
 
-
-const authServerURL = 'http://localhost:3000';
+const authServerURL = `http://${process.env.SITE_URL}:${process.env.PORT}`;
```

Comment:
This dynamically sets the URL to the data in our env file, allowing it to toggle between a live site and a local host in our dev environment